docs/src/model_setup/background_fields.md

```diff
@@ -36,6 +36,9 @@ velocity field.
 Other possible terms associated with the Coriolis force, buoyancy, turbulence closures,
 and surface waves acting on background fields are neglected.
 
+!!! compat "Model compatibility"
+    At the moment _only_ the [`NonhydrostaticModel`](@ref) supports `BackgroundFields`.
```

Comment:
```suggestion
    `BackgroundFields` are only supported by [`NonhydrostaticModel`](@ref).
```

Reply:
I feel like saying "at the moment" implies there are plans to implement support, but I don't think there are...

Reply:
True. No need to imply anything about the future. Talk about the present.